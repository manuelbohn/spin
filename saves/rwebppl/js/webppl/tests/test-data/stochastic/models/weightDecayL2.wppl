var model = function() {
  var mu = param();
  observe(Gaussian({mu, sigma: 1}), 3);
  return mu;
};
