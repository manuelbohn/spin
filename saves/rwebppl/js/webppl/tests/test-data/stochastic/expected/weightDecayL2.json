{
  "mean": 1.5
}
